src/content/en/fundamentals/media/fast-playback-with-video-preload.md

```diff
@@ -0,0 +1,571 @@
+
+If the video source is an unique file hosted on a web server, you may want to
+use the video `preload` attribute to provide a hint to the browser as to [how
+much information or content to preload]. This means [Media Source Extensions
+(MSE)] is not compatible with `preload`.
+
+Resource fetching will start only when initial HTML document has been
+completely loaded and parsed (eg. `DOMContentLoaded` event) while the very
+different `load` event will be fired when resource has actually been fetched.
+
+<figure class="TODO">
+  <img src="/web/fundamentals/media/images/video-preload/video-preload.svg">
+</figure>
+
+Setting `preload` value to `metadata` indicates that the user is not expected
+to need the video, but that fetching its metadata (dimensions, track list,
+duration, and so on) is desirable.
+
+```
+<video id="video" preload="metadata" src="file.mp4" controls></video>
+
+<script>
+  video.addEventListener('progress', function() {
+    if (video.buffered.length === 0) return;
+
+    var bufferedSeconds = video.buffered.end(0) - video.buffered.start(0);
+    console.log(bufferedSeconds + ' seconds of video are ready to play!');
+  });
+</script>
+```
+
+Setting `preload` value to `auto` indicates that the browser should cache
+enough such that the playback to end would be possible without requiring a stop
+for further buffering.
+
+```
+<video id="video" preload="auto" src="file.mp4" controls></video>
+
+<script>
+  video.addEventListener('progress', function() {
+    if (video.buffered.length === 0) return;
+
+    var bufferedSeconds = video.buffered.end(0) - video.buffered.start(0);
+    console.log(bufferedSeconds + ' seconds of video are ready to play!');
+  });
+</script>
+```
+
+There are some caveats though. As this is just a hint, browser may completely
+ignore the `preload` attribute. At the time of writing, here are some rules
+applied in Chrome:
+
+- When [Data Saver] is enabled, Chrome forces `preload` value to `none`.
+- In Android 4.3, Chrome forces `preload` value to `none`.
+- On a cellular connection (2G, 3G, and 4G), Chrome forces `preload` value to
+  `metadata`.
+
+### Tips
+
+If your website contains plenty of video resources on the same domain, I would
+recommend you set the `preload` value to `metadata` or define the `poster`
+attribute and set `preload` value to `none`. That way, you would avoid hitting
+the maxmium number of HTTP connections per host (6 in Chrome at the time of
+writing) which can hang loading of resources. Note that this may also improve
+page speed if video aren't part of your core user experience.
+
+## Link preload
+
+As [covered] in other [articles], [link preload] is a declarative fetch that
+allows you to force the browser to make a request for a resource without
+blocking the document's `onload` event. Resources loaded via `<link
+rel="preload">` are stored locally in the browser, and are effectively inert
+until they're explicitely referenced in the DOM, JavaScript, or CSS.
```

Comment:
@wolenetz Fixed.

@johnpallett I wouldn't recommend this. 

> Both prefetch ([RESOURCE-HINTS]) and preload declare a resource and its fetch properties, but differ in how and when the resource is fetched by the user agent: prefetch is an optional and low-priority fetch for a resource that might be used by a subsequent navigation; preload is a mandatory and high-priority fetch for a resource that is necessary for the current navigation. Developers should use each one accordingly to minimize resource contention and optimize load performance.

Source: https://w3c.github.io/preload/